src/views/Layers.vue

```diff
@@ -1,36 +1,62 @@
 <template>
   <v-card class="layers" flat>
-    <v-card-text>
-      Lorem ipsum dolor sit amet.
-    </v-card-text>
+    <v-select
+      v-model="selectedTags"
+      class="px-4 pt-4 pb-1"
+      label="Filter by tag"
+      multiple
+      dense
+      outlined
+      hide-details
+      :items="layerTags"
+    />
     <layer-list-controls
-      v-if="layers"
-      :layers="layers"
+      v-if="layersForTree"
+      :layers="layersForTree"
       @active-layers-change="onActiveLayerChange"
       @layer-sorting-change="onLayerSortingChange"
     />
   </v-card>
 </template>
 
 <script>
-  import { mapActions, mapState } from 'vuex'
+  import { mapActions, mapGetters } from 'vuex'
   import { LayerListControls } from '@deltares/vue-components'
 
   export default {
     name: 'Layer',
+
     components: {
       LayerListControls,
     },
+
+    data: () => ({
+      selectedTags: [],
+    }),
+
     computed: {
-      ...mapState('data', [ 'layers' ]),
+      ...mapGetters('data', [
+        'displayLayers',
+        'flattenedLayers',
+        'layerTags',
+      ]),
+
+      layersForTree() {
```

Comment:
If it turns out to be troublesome to replace the input for the layer tree, there's also a filter option in the `v-treeview` component, but it's currently not exposed by the `layer-list-controls` component